packages/sites/genomics-site/webapp/wdkCustomization/js/client/components/questions/EdaNotebookQuestionForm.tsx

```diff
@@ -0,0 +1,53 @@
+import DefaultQuestionForm, {
+  Props,
+} from '@veupathdb/wdk-client/lib/Views/Question/DefaultQuestionForm';
+import React, { useCallback } from 'react';
+import { EdaNotebookParameter } from './EdaNotebookParameter';
+import {
+  ParameterGroup,
+  Parameter,
+  ParameterValues,
+} from '@veupathdb/wdk-client/lib/Utils/WdkModel';
+
+export const EdaNotebookQuestionForm = (props: Props) => {
+  const { searchName } = props;
+  if (!searchName) {
+    throw new Error('No search defined.');
+  }
+
+  // We'll use this function throughout the notebook to update any wdk parameters.
+  const wdkUpdateParamValue = useCallback(
+    (
+      parameter: Parameter,
+      newParamValue: string,
+      paramValues: ParameterValues
+    ) => {
+      props.eventHandlers.updateParamValue({
+        searchName,
+        parameter,
+        paramValues,
+        paramValue: newParamValue,
+      });
+    },
+    [props, searchName]
+  );
+
+  // An override that renders the notebook instead of any default parameter or parameter group ui.
+  const renderParamGroup = (group: ParameterGroup, formProps: Props) => {
+    return (
+      <EdaNotebookParameter
+        value={'test'}
+        parameters={props.state.question.parameters}
+        wdkUpdateParamValue={wdkUpdateParamValue}
+      />
+    );
+  };
+
+  return (
+    <DefaultQuestionForm
+      {...props}
+      renderParamGroup={renderParamGroup}
```

Comment:
This relies on there only being **one** non-hidden paramGroup in the question. If there were two, we'd get two similar-looking EDA notebooks.
Here's the relevant code from `DefaultQuestionForm` L207
```
            {question.groups
              .filter((group) => group.displayType !== 'hidden')
              .map((group) => renderParamGroup(group, props))}
```

If this works in practice for now, no biggie, but it's something to consider in the future.